docs/login-testcase.md

```diff
@@ -0,0 +1,21 @@
+# Login testcase
+
+## Prerequisites
+User has acount in the system already created by system admin.
+
+## Steps
+
+### Step 1
+User opens pah-fm website.
```

Comment:
At some point we would need to distinguish instance address (eg. development, release candidate or production) + that we would need to have example accounts there. As a rule of thumb there are the same accounts on every single one of them aside from the passwords - that allows testers/business people to easily test something out. 

Treat it more as a general comment.